### PR TITLE
Fix perl script on examples

### PR DESCRIPTION
**Pull request by JohnTitor**
[The latest version of perl's `bignum` package has a breakage](https://github.com/pjacklam/p5-bignum/issues/2) and `bpi(2000)` returns an error. This replaces it with `bpi(1000)` as it's enough for examples.
Fixes https://github.com/kubernetes/website/issues/34073

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ spec:
       containers:
       - name: pi
         image: perl
-        command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+        command: ["perl", "-Mbignum=bpi", "-wle", "print bpi(1000)"]
```
Comment (bradtopol) [inline content/en/docs/concepts/workloads/controllers/job.md:346]: Good choice! 1000 digits of pi should be enough.  Great catch!

**Comment by bradtopol** [comment]
The change from 2000 digits of pi to 1000 digits seems appropriate.  1000 is large enough to demonstrate the concept and does not throw an error which is distracting.  Great job!

File: content/en/examples/controllers/job.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ spec:
       containers:
       - name: pi
         image: perl
-        command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+        command: ["perl", "-Mbignum=bpi", "-wle", "print bpi(1000)"]
```
Comment (bradtopol) [inline content/en/examples/controllers/job.yaml:11]: I did find one issue. If you look at https://deploy-preview-34070--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/job/ which references this example it says pi is calculated to 2000 digits.  Can you update this page to say 1000 as well?

Comment (JohnTitor) [reply content/en/examples/controllers/job.yaml:11]: Good point, fixed via f9445734525c2445e1aa16112a0e534b1f7b47d7.

**Comment by paulyeo21** [comment]
I was able to reproduce issue and confirm this works

**Comment by sftim** [comment]
Thanks for suggesting this fix. PR https://github.com/kubernetes/website/pull/34475 proposed a different fix, which I've approved.

**Comment by JohnTitor** [comment]
@sftim You prefer to pin the version over fix the root issue?

**Comment by sftim** [comment]
Pinning container image versions is definitely a good practice.

If anyone also wants to fix the bug in Perl's `bignum` library, that's fine - no objection there.

I don't think we need this PR though.

**Pull request closed without merging**